modules/gitops-release-notes-1-15-2.adoc

```diff
@@ -0,0 +1,40 @@
+
+Issued: 2025-05-16
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2025:7753[RHSA-2025:7753]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-15-2_{context}"]
+== Fixed issues
+
+* Before this update, Argo CD components, such as, `server`, `repo-server`, and `application-controller`, could crash when accessing the Redis instance due to network or DNS instabilities within the cluster. This issue stemmed from a race condition in the `go-redis` client library when multiple connections in a connection pool call the `dial hook` function. With this update, this issue is fixed. This update resolves the issue by integrating an updated `go-redis` client library that eliminates race conditions during `dial hook` function calls. It also improves the handling and recovery from network and DNS errors, ensuring greater stability for Argo CD components. link:https://issues.redhat.com/browse/GITOPS-6287[GITOPS-6287]
+
+* Before this update, upgrading the {gitops-title} Operator to v1.15.1 raised a health check error that prevented Red Hat Advanced Cluster Management (ACM) policies from syncing. This update fixes the issue by adding a missing nil check to `.status.placement` for *Policy*. link:https://issues.redhat.com/browse/GITOPS-6664[GITOPS-6664]
+
+* Before this update, when `.spec.source.targetRevision` was set to an annotated Git tag, Argo CD treated resource drift as needing reconciliation. This caused manual changes on the cluster to be reverted, despite self-healing being disabled. This behavior was inconsistent with that of lightweight tags. With this update, Argo CD now handles annotated and lightweight tags consistently. When `selfHeal` is set to `false`, manual changes to live resources are no longer reverted, even if you are using an annotated Git tag as the source. link:https://issues.redhat.com/browse/GITOPS-6150[GITOPS-6150]
+
+* Before this update, monitoring support for workloads deployed in {OCP} namespaces, prefixed with `openshift-`, was available by default. Argo CD monitoring worked automatically only when deployed in {OCP} managed namespaces. With this update, monitoring support for Argo CD deployed in non-{OCP} namespaces requires monitoring for user-defined projects to be explicitly enabled in {OCP} Monitoring. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/monitoring/configuring-user-workload-monitoring#configurable-monitoring-components_preparing-to-configure-the-monitoring-stack-uwm[Enabling monitoring for user-defined projects]. link:https://issues.redhat.com/browse/GITOPS-6251[GITOPS-6251]
```

Comment:
> Before this update, monitoring support for workloads deployed in {OCP} namespaces, prefixed with `openshift-`, was available by default. 

This was the case even for non-prefixed ns, right?

> Argo CD monitoring worked automatically only when deployed in {OCP} managed namespaces.

Same as above, this case is after the upgrade.

Could you please clarify, Anand?

Reply:
Fixed.